### ISSUE TITLE
Make integration tests faster (eg. finished in less than 3 minutes)

### ISSUE THREAD

**Issue by suzaku** [pingcap/tidb-binlog#582]
The task of running integration tests is the bottleneck of the build pipeline.
We may try to make it faster by:

1. Reducing the number of SQLs tested, may be some of the tests should belong to chaos testing instead of integration testing;
1. Split the task into multiple ones and run them in parallel

**Comment by july2993** [comment]
3 minute..?
i think we can first make the test case in `tests/dailytest` run in parallel.

**Comment by suzaku** [comment]
@july2993 Yes, @jebter mentioned that some other projects in our team are limiting the run time of integration tests to 3 minute by running them in parallel.
